docs/source/development/reference/functions.rst

```diff
@@ -751,7 +751,26 @@ Distances
               and the third value is the reverse azimuth. The fourth coordinate
               value is unused.
 
+.. c:function:: PJ_COORD proj_geod_direct(const PJ *P, PJ_COORD a, double azimuth, double distance)
 
+    Solves the direct geodesic problem for given projection ellipsoid
+
+    The coordinates in :c:data:`a` needs to be given as longitude and latitude in 
+    radians. Note that the axis order of the :c:data:`P` object is not taken into 
+    account in this function, so even though a CRS object comes with axis ordering
+    latitude/longitude coordinates used in this function should be reordered as 
+    longitude/latitude. The azimuth  :c:data:`azimuth` should be provided in radians,
+    and :c:data:`distance` should be provided in meters
+
+    :param P: Transformation or CRS object
+    :type P: const :c:type:`PJ` *
+    :param PJ_COORD a: Coordinate of first point
+    :param double azimuth: Initial bearing from the starting point in radians
```

Comment:
it would be useful to specify what the axis from which the angle is measured is, and if it is clockwise or counter-clockwise